src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenScriptTests.cs

```diff
@@ -525,16 +525,17 @@ .maxstack  2
             Assert.True(methodData.Method.ReturnsVoid);
             methodData.VerifyIL(
 @"{
-  // Code size       24 (0x18)
+  // Code size       25 (0x19)
   .maxstack  1
-  .locals init (System.Runtime.CompilerServices.TaskAwaiter V_0)
+  .locals init (System.Runtime.CompilerServices.TaskAwaiter<object> V_0)
   IL_0000:  newobj     "".ctor()""
   IL_0005:  callvirt   ""System.Threading.Tasks.Task<object> <Initialize>()""
-  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter System.Threading.Tasks.Task.GetAwaiter()""
+  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter<object> System.Threading.Tasks.Task<object>.GetAwaiter()""
```

Comment:
Do you mean the fact that we are now calling a different API? That's an implementation detail of scripting that we don't guarantee is gonna stay the same.

---
In reply to: [119451463](https://github.com/dotnet/roslyn/pull/19623#discussion_r119451463) [](ancestors = 119451463,119436197)